python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -69,6 +69,12 @@ def __init__(self, torch_feature: torch.Tensor):
             f"dimension of torch_feature in TorchBasedFeature must be greater "
             f"than 1, but got {torch_feature.dim()} dimension."
         )
+        if not torch_feature.is_contiguous():
```

Comment:
Is this necessary? I suspect `contiguous` will check it inside.

Reply:
Yes, I do want to alert the user to the weird phenomenon, so they will check their own dataset. Otherwise, the user may never realize that their features aren't contiguous.